app/Models/ApplicationPreview.php

```
@@ -50,14 +67,72 @@ public function application()
+            $schema = $url->getScheme();
+            $random = new Cuid2;
+            $preview_fqdn = str_replace('{{random}}', $random, $template);
+            $preview_fqdn = str_replace('{{domain}}', $host, $preview_fqdn);
+            $preview_fqdn = str_replace('{{pr_id}}', $this->pull_request_id, $preview_fqdn);
+            $preview_fqdn = "$schema://$preview_fqdn";
+            $this->fqdn = $preview_fqdn;
+            $this->save();
+        }
+
+        return $this;
+    }
```

PHPMD (2.15.0)

The method generate_preview_fqdn is not named in camelCase. (Controversial Rules)

(CamelCaseMethodName)